server/src/completions/spdx-licenses.ts

```diff
@@ -54,21 +54,27 @@ const spdxLicensesFileContent = fs.readFileSync(spdxLicensesPath, 'utf8')
 export const spdxLicensesCollection = JSON.parse(spdxLicensesFileContent) as SpdxLicenseCollection
 export const spdxLicenses = spdxLicensesCollection.licenses
 
-export const licenseCompletionItems: CompletionItem[] = spdxLicenses.map((license) => (
-  {
-    label: license.licenseId,
-    kind: CompletionItemKind.Value,
-    deprecated: license.isDeprecatedLicenseId === true,
-    labelDetails: {
-      description: spdxLicenseDescription
-    },
-    documentation: 'Loading...',
-    data: {
-      source: spdxLicenseDescription,
-      payload: license
-    } satisfies CompletionItemData
-  }
-))
+export const getLicenseCompletionItems = (rangeOfText: Range): CompletionItem[] => {
+  return spdxLicenses.map((license) => (
```

Comment:
Since this is fully implemented on the server side, it should work with neoVim. Can you test it? Also to ensure it didn't break the npm packaging :stuck_out_tongue:

Reply:
That one gives me lot of trouble. Spent too much time today trying to launch the language server on neovim :/

Reply:
Currently not working on NVIM, there was actually a regression which I fix in: !tree-sitter-bitbake.wasm

I don't know why the licenses don't show with NVIM, the package .tar.gz looks good. It would be nice to investigate. Tell me what you couldn't get working with NVIM. The documentation should be updated if something is missing.

Reply:
Done.

I am pointing out the use of node-cache in order to eventually release the memory. I wonder if there are other places we could consider using it.